index.md

```diff
@@ -13,9 +13,14 @@ Fabric is a lightweight, experimental modding toolchain for Minecraft.
 
 To install Fabric, you need the **Fabric Loader**, which you can <a class="page-link page-link-download" href="/use">download here</a>
 
-In addition, most mods depend on an API mod, containing commonly used hooks. They can be found here:
+You should also read the [Fabric Wiki installation instructions](https://fabricmc.net/wiki/install)!
 
-* [Fabric API for Minecraft 1.14 and above](https://minecraft.curseforge.com/projects/fabric/files)
+In addition, most mods depend on the Fabric API mod, containing commonly used hooks. It must be added to your mods folder. Download Fabric API for Minecraft 1.14 and above from here:
+
+* [CurseForge download for Fabric API](https://minecraft.curseforge.com/projects/fabric/files)
+* [Modrinth download for Fabric API](https://modrinth.com/mod/fabric-api/versions)
+
+It is recommended that you read the <a href="https://fabricmc.net/wiki/faq:user">Frequently Asked Questions for users.</a> 
```

Comment:
Why use an `<a>` tag here, and mark down above?

Reply:
Oops! Would be great if you could make these changes. I'm going to have to go now xD

Reply:
No worries, I think I will want to checkout these changes first and double check they look good on a built site. Its quite hard to review/visualise HTML 👍